<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCColonos</class>
 <widget class="QWidget" name="DCColonos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightblue</string>
  </property>
  <widget class="QLabel" name="separador1">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>241</width>
     <height>631</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>520</y>
     <width>731</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="separador3">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="carta_arcilla">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>71</width>
     <height>103</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_arcilla">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_madera">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>71</width>
     <height>103</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_madera">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_trigo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>71</width>
     <height>103</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_trigo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="dado_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dado_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>769</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>Turno de:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_nombre_turno">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_contrincante_1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>Puntos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_puntos_contrincante_1">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_arcilla_1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>120</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_madera_1">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>120</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_trigo_1">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_arcilla_1">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>140</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_madera_1">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>140</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_trigo_1">
   <property name="geometry">
    <rect>
     <x>955</x>
     <y>140</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_madera_2">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>255</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_madera_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>270</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_arcilla_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>255</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_trigo_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>255</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_trigo_2">
   <property name="geometry">
    <rect>
     <x>955</x>
     <y>270</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_arcilla_2">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>270</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_puntos_contrincante_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>220</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>Puntos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_contrincante_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_arcilla_3">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>400</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_contrincante_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>350</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_madera_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>400</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_trigo_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>385</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_trigo_3">
   <property name="geometry">
    <rect>
     <x>955</x>
     <y>400</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>350</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>Puntos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_puntos_contrincante_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>350</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_madera_3">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>385</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_arcilla_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>385</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>Carretera más larga:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_nombre_carretera_larga">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>510</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_nombre_cliente">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="carta_puntos_victoria">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>570</y>
     <width>41</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_puntos_victoria">
   <property name="geometry">
    <rect>
     <x>804</x>
     <y>590</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>590</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>Puntos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable_puntos_cliente">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>590</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>569</y>
     <width>5</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>569</y>
     <width>5</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>569</y>
     <width>5</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgrey</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
